src/itemset.js

```diff
@@ -180,7 +222,8 @@ define([
         },
         updateControlNodeAdaptorMap: function (map) {
             this.__callOld();
-            var adaptItemset = parser.makeControlOnlyMugAdaptor('Itemset');
+            var adaptItemset = parser.makeControlOnlyMugAdaptor('Itemset'),
+                use_advanced_itemsets = this.opts().features.advanced_itemsets;
```

Comment:
`updateControlNodeAdaptorMap` is called by `parser.init()`, which is called by `core.postInit`, and that should be called after all plugins have been initialized. So I'm not sure why this particular one is a problem.

On the other hand, `getMugTypes` is called by `core.init()`, which is called before any other plugins are initialized. That one is a problem for you here. It may be worth investigating if we can replace [these lines](https://github.com/dimagi/Vellum/blob/f93eb0f304c088e976437dde702e953a519e278c/src/base.js#L144-L149) with simply `instance.init()`, which would init all other plugins before core (in the reverse order from how they are currently initialized)—this seems pretty elegant to me and may be worth some effort. Another option would be to move the `getMugTypes` call from `core.init()` to `core.postInit()`.